src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolverFactory.cs

```diff
@@ -1,14 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.ComponentModel.Composition;
+using System.Composition;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Razor;
 
 namespace Microsoft.VisualStudio.LanguageServices.Razor
 {
-    [ExportLanguageServiceFactory(typeof(TagHelperResolver), RazorLanguage.Name, ServiceLayer.Default)]
+    [Shared]
+    [ExportLanguageServiceFactory(typeof(TagHelperResolver), RazorLanguage.Name, ServiceLayer.Host)]
     internal class DefaultTagHelperResolverFactory : ILanguageServiceFactory
```

Comment:
This isn't the default one anymore, this is VS (Windows) specific.

Reply:
Isn't that portrayed by the assembly?